### PR TITLE
fix: don't create styled component in render function

### PR DESCRIPTION
**Pull request by brandonc**
There are a few more examples of this in Sidebar and Headerbar, but this in particular is much easier to fix and is affecting protoss performance

### CODE REVIEW

**Comment by brandonc** [comment]
@jaxesn had to update a snapshot

File: packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap [hunk 1]
```diff
@@ -259,19 +259,9 @@ exports[`Dropdownable is visible after opening 2`] = `
...
-  left: 0px;
-  position: absolute;
-  -webkit-transform: scale(1);
-  -ms-transform: scale(1);
-  transform: scale(1);
-  opacity: 1;
-}
-
-<div>
+            <div>
```
Comment (jaxesn) [inline packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap]: I dont understand why this changed?  This looks like an unrelated class?

Comment (brandonc) [reply packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap]: I was surprised at first, but if styled was re-creating a component every render, it could be that it creates these styles each render as well, which was happening multiple times in this test. Now that the component is only created once, there's less style code created?

Comment (brandonc) [reply packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap]: I'm not too familiar with what emotion does under the hood

Comment (jaxesn) [reply packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap]: @mperrotti we might need you to weigh in on this

Comment (mperrotti) [reply packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap]: @brandonc's assessment sounds correct to me
